pmlc/rt/vulkan/tests/BUILD

```diff
@@ -2,8 +2,31 @@
 
 package(default_visibility = ["//visibility:public"])
 
-load("//pmlc:lit.bzl", "glob_lit_tests")
+load("//pmlc:lit.bzl", "glob_lit_tests", "lit_test")
+load("//bzl:plaidml.bzl", "plaidml_cc_test")
 
 glob_lit_tests(
     data = ["//pmlc/rt/vulkan:vkprobe"],
 )
+
+plaidml_cc_test(
+    name = "cc_test_gpu",
+    srcs = ["edsl_test_gpu.cc"],
```

Comment:
Could we turn `edsl_test.cc` into a library so that we can use it for both CPU and GPU tests?

Reply:
I would like to. But for now, one thing is the PLAIDML_DEVICE and PLAIDML_TARGET are hardcoded in the env, we might need two testenv.cc. Another thing is many tests can pass now but there are still a few tests fails on gpu(which I commented in the file) due to limited support in mlir passes (like spv pointer to int1 is not supported, unnested loop et al). In addition, last time I saw the Linux CI server fails for some Uint64 tests, so I commented the Uint64 tests. Also some tests commented in cc_tests.cc passed in cc_test_gpu.cc like GradientDot.
So the passed test set is slightly different for gpu and cpu right now.

Reply:
Could you please suggest how can I set PLAIDML_DEVICE and PLAIDML_TARGET to intel_gen for this test? I am considering adding a copy of testenv.h testenv.cc to pmlc/rt/vulkan/tests for cc_test_gpu only.